src/radical/pilot/controller/pilot_launcher_worker.py

```diff
@@ -306,6 +306,19 @@ def run(self):
 
                 if self._disabled.is_set():
                     # don't process any new pilot start requests.  
+                    # NOTE: this is not clean, in principle there could be other
+                    #       launchers alive which want to still start those 
+                    #       pending pilots.  In practice we only ever use one
+                    #       pmgr though, and its during its shutdown that we get
+                    #       here...
```

Comment:
No, I don't think so.  But in principle two launch controllers could be used for one pmgr, and this commit explicitly breaks that.  Well, OTOH, this is already broken in other places, so it doesn't change much :/